docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,11 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
+         - Dropwizard transitive dependencies have been removed from the ``atlasdb-config`` subproject.
+           Configuration deserialization no longer utilizes Dropwizard's `DiscoverableSubtypeResolver <https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jackson/src/main/java/io/dropwizard/jackson/DiscoverableSubtypeResolver.java>`__.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3218>`__)
```

Comment:
We should update the above to say something like `We still support discovering subtypes of config, as we ship AtlasDB with an implementation of Dropwizard's DiscoverableSubtypeResolver` after the change we kept in. 

It is a dev break for people who relied on the transitive dependency, but for what I hope the common case is (i.e. if you need dropwizard you pull it in yourself directly!) the latest version of this change shouldn't break it